### PR TITLE
Bump every package to version 0.2

### PR DESCRIPTION
**Pull request by nnmm**
The reason for the "bump" is to have a consistent version numbering across packages, not because it's needed for a release.

### CODE REVIEW

**Comment by jhdcs** [comment]
This appears good to me! The version "regression" in `rclrs_examples` is just a correction, right?

**Review by jhdcs — APPROVED** [review]

**Review by esteve — CHANGES_REQUESTED** [review]
Leaving a comment here before this gets merged and I'll add more later, but this PR needs to be split into multiple ones and not ok with the author field. The tradition in ROS is to keep the author field intact, but add maintainers. For example, rclcpp still has Dirk as the sole author, despite William being the most active committee and me being the third. Nevertheless, Jacob should be added as a maintainer as well.

**Comment by nnmm** [comment]
Ok, sure @esteve. I wasn't clear about the meaning of the author field. I'm making this PR just about the version numbers then.

**Comment by esteve** [comment]
@nnmm thanks!

**Comment by nnmm** [comment]
@esteve Done.

**Comment by nnmm** [comment]
@esteve Could you approve if no further changes are required?

**Review by esteve — APPROVED** [review]

**Pull request merged**
